packages/graphql/src/constants.ts

```diff
@@ -34,3 +34,27 @@ export const REQUIRED_APOC_PROCEDURES = ["apoc.util.validate", "apoc.do.when", "
 export const DEBUG_AUTH = `${DEBUG_PREFIX}:auth`;
 export const DEBUG_GRAPHQL = `${DEBUG_PREFIX}:graphql`;
 export const DEBUG_EXECUTE = `${DEBUG_PREFIX}:execute`;
+
+// [0]Property [1]Error
+export const RESERVED_NODE_LABELS = [
+    [
+        "PageInfo",
+        "Type name `PageInfo` reserved to support the pagination model of connections. See https://relay.dev/graphql/connections.htm#sec-Reserved-Types for more information.",
+    ],
+    [
+        "Connection",
+        'Type names ending "Connection" are reserved to support the pagination model of connections. See https://relay.dev/graphql/connections.htm#sec-Reserved-Types for more information.',
+    ],
+    ["Node", "Type name 'Node' reserved to support relay See https://relay.dev/graphql/"],
+];
+
+// [0]Property [1]Error
+export const RESERVED_INTERFACE_NAMES = [
+    ["Node", "Interface name 'Node' reserved to support relay See https://relay.dev/graphql/"],
+];
+
+// [0]Property [1]Error
+export const RESERVED_INTERFACE_PROPERTIES = [
```

Comment:
`RESERVED_FIELD_NAMES`? To avoid the leakage of "properties". Like, you could have a property "node" when we introduce aliasing, it's just GraphQL field names that are the problem.